public/css/modals/application-review-modal.css

```diff
@@ -68,6 +64,13 @@
 	margin: 4px 0px 16px;
 }
 
+/* Gets rid of giant empty black space
+   at bottom of last application content */
+.black-tab-container::after {
+	display: none;
+	height: 0;
+}
+
```

Comment:
Unfortunately, this means the spacing is different between white and black sections. What really creates the massive extra space is the combination of this `::after` and also the `padding-bottom` in `.application-content`. I'd say the easier (and more consistent) thing to do is to remove the `padding-bottom` and reinstate the `::after`s.

If you want to remove just the last `::after`, check out the [`:nth-last-child()`](https://www.w3schools.com/cssref/sel_nth-last-child.asp) selector.

Reply:
Awesome, that worked! I combined the `last-child` and `::after` selectors to get rid of the black space at the bottom. Thanks for catching this!